packages/editor/src/store/reducer.js

```diff
@@ -87,29 +88,52 @@ function mapBlockOrder( blocks, rootClientId = '' ) {
 }
 
 /**
- * Given an array of blocks, returns an object containing all blocks, recursing
- * into inner blocks. Keys correspond to the block client ID, the value of
- * which is the block object.
+ * Helper method to iterate through all blocks, recursing into inner blocks,
+ * applying a transformation function to each one.
+ * Returns a flattened object with the transformed blocks.
  *
- * @param {Array} blocks Blocks to flatten.
+ * @param {*} blocks Blocks to flatten.
+ * @param {*} transform Transforming function to be applied to each block.
```

Comment:
It's not a wildcard `*`, it's a `Function`.

Reply:
👍